client/blocks/reader-combined-card/index.jsx

```diff
@@ -176,8 +176,14 @@ function combinedCardPostKeyToKeys( postKey ) {
 export default connect( ( state, ownProps ) => {
 	const postKeys = combinedCardPostKeyToKeys( ownProps.postKey );
 
+	// Posts are directly supplied in Devdocs
+	let posts = ownProps.posts;
+	if ( ! posts ) {
+		posts = getPostsByKeys( state, postKeys );
+	}
```

Comment:
for funsies we can also remove the mutation and express this inline…

```js
const posts = ownProps.posts
	? ownProps.posts
	: getPostsByKeys( state, postKeys );
```

Reply:
further since we're only wanting the first of many possible assignments we can remove the ternary and even inline this below where it was originally `getPostsByKeys( state, postKeys )`

```js
posts: ownProps.posts || getPostsByKeys( state, postKeys ), // provided posts take precedence over state posts
```

Reply:
👍 I'm going to take your advice below and use the un-`connect`-ed component in Devdocs.